### PR TITLE
feat(package): add repository information to package.json

### PR DESCRIPTION
**Pull request by Aniruddha-Shriwant**
Signed-off-by: Aniruddha Shriwant <aniruddhashriwant@gmail.com>

# Closes #[341](https://github.com/accordproject/techdocs/issues/341)<CORRESPONDING ISSUE NUMBER>

### Changes

-  Added Package Repository Information (directory) in `package.json` <ONE>

### Related Issues
- Issue #<NUMBER>
- Pull Request #243 <NUMBER>

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`
- [ ] Manual accessibility test performed
    - [ ] Keyboard-only access, including forms
    - [ ] Contrast at least WCAG Level A
    - [ ] Appropriate labels, alt text, and instructions

### CODE REVIEW

File: packages/storybook/package.json
Comment (irmerk) [inline packages/storybook/package.json:67]: Similar to the dingus in `markdown-transform`, I don't think we want to bother with `storybook`

**Review by DianaLease — APPROVED** [review]
LGTM

File: packages/storybook/package.json
Comment (Aniruddha-Shriwant) [reply packages/storybook/package.json:67]: So should I revert changes for this file?

**Comment by irmerk** [comment]
Please don't merge until all related PRs are approved (tracked in https://github.com/accordproject/techdocs/issues/341)

**Review by dselman — APPROVED** [review]
LGTM!

**Pull request merged**
